package com.github.dingey.auth.exception;

public class AuthException extends RuntimeException {
    public AuthException(Throwable cause) {
        super(cause);
    }
}
